src/Middleware/OutputCaching/src/OutputCacheMiddleware.cs

```diff
@@ -336,10 +336,23 @@ internal async Task<bool> TryServeFromCacheAsync(OutputCacheContext cacheContext
         // Locking cache lookups by default
         // TODO: should it be part of the cache implementations or can we assume all caches would benefit from it?
         // It makes sense for caches that use IO (disk, network) or need to deserialize the state but could also be a global option
+        OutputCacheEntry? cacheEntry;
+        try
+        {
+            cacheEntry = await _outputCacheEntryDispatcher.ScheduleAsync(cacheContext.CacheKey, (Store: _store, CacheContext: cacheContext), static async (key, state) => await OutputCacheEntryFormatter.GetAsync(key, state.Store, state.CacheContext.HttpContext.RequestAborted));
+        }
+        catch (OperationCanceledException)
+        {
+            // don't report as failure
+            cacheEntry = null;
+        }
+        catch (Exception ex)
+        {
+            _logger.LogError(ex, "Unable to query output-cache");
```

Comment:
Use the same pattern as other logs, i.e. `[LoggerMessage(...)]`.

And end log in `.`